tide_site.module

```diff
@@ -576,27 +580,45 @@ function tide_site_taxonomy_term_delete($term) {
 /**
  * Implements hook_ENTITY_TYPE_update().
  */
-function tide_site_node_update($node) {
-  // Regenerate site aliases of the updated node.
-  /** @var \Drupal\tide_site\AliasStorage $alias_storage */
+function tide_site_node_update(NodeInterface $node) {
+  // Gets required services.
   $alias_storage_helper = \Drupal::service('tide_site.alias_storage_helper');
-
+  $tide_site_helper = \Drupal::service('tide_site.helper');
   $path_storage = \Drupal::entityTypeManager()->getStorage('path_alias');
 
-  /** @var \Drupal\node\NodeInterface $node */
+  // If the node has no aliases, we should creat them.
   if (!$alias_storage_helper->loadAll(['path' => '/node/' . $node->id()])) {
-    /** @var \Drupal\pathauto\PathautoGenerator $path_auto_generator */
     $path_auto_generator = \Drupal::service('pathauto.generator');
     $path_auto_generator->createEntityAlias($node, 'update');
   }
+
+  // Get last node revision for comparison purpose.
   $old_node = $node->original;
-  $helper = \Drupal::service('tide_site.helper');
-  $old_sites = $helper->getEntitySites($old_node, TRUE) ?: ['ids' => []];
-  $new_sites = $helper->getEntitySites($node, TRUE) ?: ['ids' => []];
+  // If node title changed, we should change the aliases accordingly.
```

Comment:
There's a fair bit of logic getting thrown in here, it's probably worth considering splitting it out to a service.

Reply:
Thanks @GROwen , yes.  too much code resides in tide_site.module file. I will try to make it clearer to understand and easy to read.